### ISSUE TITLE
unable to clear current value

### ISSUE THREAD

**Issue by juliangruber** [hypergraph-xyz/cli#14]
Due to https://github.com/terkelg/prompts/issues/219 the user currently can't use the update _prompt_ to clear a value (like `description`) because the current value will always be used instead. Clearing a value via `update <key> <value>` does work however.

**Comment by jameslibscie** [comment]
@juliangruber Is this issue basically blocked until terkelg/prompts#219 is fixed?

**Comment by juliangruber** [comment]
Yes this is blocked until that is fixed, however we can contribute the fix ourselves if we feel the need.
